### PR TITLE
FTP server with anonymous access, providing a unified view of multiple content directories

### PR DESCRIPTION
**Pull request by goodbadwolf**
This PR adds code for the base functionality for LFTP. 
It adds an FTP server which provides a read-only access for content, which provides a unified view for multiple basepaths, along with a simple socket IPC based API to enable/disable the FTP server.

### CODE REVIEW

File: lftp/config.ini [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# https://docs.python.org/2/library/logging.config.html
+
+# Log format
+format = [%(asctime)s] %(levelname)-8s %(message)s
+
+# Date and time formatting
+date_format = %Y-%m-%d %H:%M:%S (%z)
+
+# Maximum size of the log file
+size = 10MB
```
Comment (foxbunny) [inline lftp/config.ini]: Drop this to 5MB and 2 backups please.

File: lftp/config.ini [hunk 2]
```diff
@@ -6,7 +6,8 @@ debug = false
 [ftp]
 host = 127.0.0.1
 port = 21
-content_path = /tmp
+basepaths =
```
Comment (foxbunny) [inline lftp/config.ini:17]: Does this mean it is completely independent of FSAL?

**Comment by goodbadwolf** [comment]
Test comment. Please ignore.

File: lftp/config.ini (hunk 2 above)
Comment (goodbadwolf) [inline lftp/config.ini:17]: Correct. No waiting for indexing. Almost-Pure FTP™ goodness

**Comment by foxbunny** [comment]
README should provide some information on what the JSON settings file contains and its format.

**Comment by foxbunny** [comment]
Also, weird version number in the setup.py. Please use 1.0.dev1 as the starting point.

**Pull request merged**